network/src/peer_store/sqlite/peer_store.rs

```diff
@@ -370,8 +370,9 @@ impl PeerStore for SqlitePeerStore {
     }
 
     fn random_peers(&self, count: u32) -> Vec<PeerAddr> {
+        const EXTRA_PEERS: u32 = 2;
```

Comment:
What's the `EXTRA_PEERS`?

Reply:
Removed, we do not actually need this now